src/app/shell/personal-cabinet/parent/parent-applications/parent-applications.component.ts

```diff
@@ -64,12 +72,22 @@ export class ParentApplicationsComponent extends CabinetDataComponent implements
    * @param Application event
    */
   onLeave(application: Application): void {
-    const applicationUpdate = new ApplicationUpdate(application.id, Statuses.Left);
-    this.store.dispatch(new UpdateApplication(applicationUpdate));
+    const dialogRef = this.matDialog.open(ConfirmationModalWindowComponent, {
+      width: Constants.MODAL_SMALL,
+      data: {
+        type: ModalConfirmationType.leaveWorkshop,
+      },
+    });
+    dialogRef.afterClosed().subscribe((result: string) => {
```

Comment:
unsubscribe pls

Reply:
@khrMykhailo  You don't need to unsubscribe as the observable itself completes.You can verify the same by adding a finalize block to see whether observable completes itself or not.